.github/workflows/test.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         env:
           - TEST: pulp
           - TEST: docs
+          - TEST: new_docs
```

Comment:
Didn't we agree to run the new docs within the "docs" runner? What changed in the meantime?

Reply:
The changes here look different:
https://github.com/pulp/plugin_template/pull/853